### ISSUE TITLE
Accessibility - provide WAI-ARIA properties for list elements

### ISSUE THREAD

**Issue by jwahyoung** [aurelia/ui-virtualization#16]
Accessibility for a virtual list could be lacking. ARIA provides a way to mitigate this; even if it may not be supported by accessibility user agents, it's worth having and doesn't break anything else.

See http://www.w3.org/TR/wai-aria/states_and_properties#aria-setsize - by defining setSize and posInset, an accessibility user agent can know how large the list is despite the entire representation not being in the DOM.

This should only require aria properties on the list items in the html template, but it may also require the `ul` element to be an aria live region.

**Comment by colin-umn** [comment]
I see this has been open for over a year.  Does that indicate a general lack of interest in Aurelia development for accessibility support?  I have been considering Aurelia for a development project for a public institution but will need to return to more established frameworks if accessibility is a big gap.

**Comment by AStoker** [comment]
@colin-umn, if you have experience with accessibility, I'd greatly appreciate a PR to help solve this issue. I don't think accessibility will be a bottle neck for you though, since you're able to write your own ARIA tags and support as much accessibility requirements as you need. Accessibility should be up to the coder, since who knows how somebody is going to use a particular plugin.   

I think for this issue, you should be able to accomplish this already. You can set attributes on an element for a virtual list like normal. I think in this particular setup if you're worried about position in sets for ARIA tags, you should be able to determine the index of the array and bind your ARIA tag to that. @jedd-ahyoung, can you give an example of how accessibility is _not_ achievable with using virtual lists?

**Comment by bigopon** [comment]
Doc links related to `aria-setsize` seem to be outdated / inactive. I couldn't find much info about it. Maybe we should close this for now. @jedd-ahyoung Do you still have any reference to an article about what mentioned?
